example-queries/sequencing-reads-query-id-fe.graphql

```diff
@@ -0,0 +1,7 @@
+query DiscoveryViewFCSequencingReadsQuery(
+    $input: queryInput_fedSequencingReads_input_Input
+) {
+    fedSequencingReads(input: $input) {
+        id
+    }
+}
```

Comment:
Do we care about ending files with a newline?

Reply:
For these tests it should be ok, we're stripping all whitespace anyways